azurerm/internal/services/sql/client.go

```diff
@@ -2,17 +2,19 @@ package sql
 
 import (
 	"github.com/Azure/azure-sdk-for-go/services/preview/sql/mgmt/2015-05-01-preview/sql"
+	vulnerabilitySvc "github.com/Azure/azure-sdk-for-go/services/preview/sql/mgmt/2017-03-01-preview/sql"
```

Comment:
I believe we have and updated version of the API already included under mssql service package, will that one work or do we need to use a 3rd?

Reply:
The `DatabaseVulnerabilityAssessmentRuleBaselinesClient` is only available in the 2017-03-01 version of the api. In the updated one there is just the managed version of this client.

See the REST API reference of Azure:
[Database Vulnerability Assessment Rule Baselines](https://docs.microsoft.com/en-us/rest/api/sql/databasevulnerabilityassessmentrulebaselines)
[Managed Database Vulnerability Assessment Rule Baselines](https://docs.microsoft.com/en-us/rest/api/sql/manageddatabasevulnerabilityassessmentrulebaselines)

Is it okay for you to leave the client in this file or should we move it to the mssql service package?

Reply:
well that is.. unfortunate -_- I presume the manage version is for managed instances?

If you don't mind moving it mssql is the place it should live as we are going to slowly retire the sql package over time as we udpate the resources to use the newer API.

Reply:
Yes, it's the version for managed instances. I moved it to the mssql clients.